docs/installation.md

```diff
@@ -1,12 +1,20 @@
 # NGI-RNAseq Installation
 
-### NextFlow installation
-See https://github.com/SciLifeLab/NGI-NextflowDocs for instructions on how to install and configure
-Nextflow.
+To start using the NGI-RNAseq pipeline, there are three steps described below:
 
-### Pipeline installation
-This pipeline itself needs no installation - NextFlow will automatically fetch it from GitHub when run if
-`SciLifeLab/NGI-RNAseq` is specified as the pipeline name.
+1. [Install Nextflow](#install-nextflow)
+2. [Install the pipeline](#install-the-pipeline)
+3. Configure the pipeline
+    * [Swedish UPPMAX System](#31-configuration-uppmax)
+    * [Other Clusters](#32-configuration-other-clusters)
+    * [Docker](#33-configuration-docker)
+    * [Amazon AWS](#34-configuration-amazon-ec2)
+
+## 1) Install NextFlow
```

Comment:
How about having something like this here;
https://github.com/IARCbioinfo/needlestack/blob/master/README.md#quick-start
```
curl -fsSL get.nextflow.io | bash
And move it to a location in your $PATH (/usr/local/bin for example here):

sudo mv nextflow /usr/local/bin
```